### PR TITLE
Add snapshot docs

### PR DESCRIPTION
**Pull request by codeliner**

### CODE REVIEW

**Comment by codeliner** [comment]
ping @prolic

File: docs/snapshots.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Snapshots
+
+One of the counter-arguments against Event-Sourcing you might heard about is that replaying events takes to much time to reconstitute an aggregate.
```
Comment (prolic) [inline docs/snapshots.md]: to much => too much

**Comment by prolic** [comment]
One small spelling, fix it, then you can merge yourself.

**Comment by codeliner** [comment]
@prolic thx for CR

**Pull request merged**